extensions/notebook/src/dialog/managePackages/addNewPackageTab.ts

```diff
@@ -7,9 +7,10 @@ import * as nls from 'vscode-nls';
 import * as azdata from 'azdata';
 import * as request from 'request';
 
-import JupyterServerInstallation from '../../jupyter/jupyterServerInstallation';
+import JupyterServerInstallation, { PipPackageOverview } from '../../jupyter/jupyterServerInstallation';
```

Comment:
Could we un-default export the JupyterServerInstallation so it's all inside the `{ }` part? Always looks weird otherwise.